src/app.ts

```diff
@@ -16,12 +17,20 @@ console.log(`${chalk.blue('i')} Using GAME_MASTER version ${chalk.cyan(gameMaste
 new PokemonWriter().Write().then(() => {
     console.log(`${chalk.green('✓')} Pokemon written to ${chalk.cyan(APP_SETTINGS.POKEMON_FILE)}`);
 }, (err) => {
+    console.log(`${chalk.red('×')} Failed at parsing Pokemon`);
     console.log(err);
-    console.log(`${chalk.red('×')} Failed at Parsing Pokemon`);
 });
 
 new MoveWriter().Write().then(() => {
     console.log(`${chalk.green('✓')} Moves written to ${chalk.cyan(APP_SETTINGS.MOVE_FILE)}`);
-}, () => {
-    console.log(`${chalk.red('×')} Failed at Parsing Moves`);
+}, (err) => {
+    console.log(`${chalk.red('×')} Failed at parsing Moves`);
+    console.log(err);
```

Comment:
Collect errors/status in a variable, and make sure the status of all operations is always placed after (or before) the errors is one option.